h2o-docs/src/product/data-science/glm.rst

```diff
@@ -229,6 +233,10 @@ Defining a GLM Model
 
 -  `export_checkpoints_dir <algo-params/export_checkpoints_dir.html>`__: Specify a directory to which generated models will automatically be exported.
 
+- **dispersion_epsilon**: If changes in dispersion parameter estimation is smaller than ``dispersion_epsilon``, will break out of the dispersion parameter estimation loop using maximum likelihood Defaults to ``0.0001``.
+
```

Comment:
For dispersion_epsilon:  I just added something:

If changes in dispersion parameter estimation or loglikelihood value is smaller than
               dispersion_epsilon, will break out of the dispersion parameter estimation loop using maximum likelihood.
               Defaults to ``0.0001``.